org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/refactoring/nls/changes/NLSChangesMessages.java

```diff
@@ -16,7 +16,7 @@
 
 public final class NLSChangesMessages extends NLS {
 
-	private static final String BUNDLE_NAME= "org.eclipse.jdt.internal.corext.refactoring.nls.changes.NLSChangesMessages";//$NON-NLS-1$
+	private static final String BUNDLE_NAME = "org.eclipse.jdt.ls.core.internal.corext.refactoring.nls.changes.NLSChangesMessages";//$NON-NLS-1$
```

Comment:
why need copy it to jdt.ls?

Reply:
It has been there, and has been copied to jdt.ls long before. And the corresponding class in jdt.ui seems inaccessible. 
Here I edit this simply because of lacking required properties. Do I have to refactor jdt.ui for it, and update in this PR?